data/human/human missions.txt

```diff
@@ -6301,3 +6301,125 @@ mission "Blind Man from Martini"
 				clear "Blind Man from Martini: label looking"
 			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`
 
+
+mission "Derelict Informer"
+	minor
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		not attributes "uninhabited"
+		not system "Rutilicus"
+	to offer
+		random < 4
+	on offer
+		conversation
+			`As you touch down on <origin>, you see that the landing pad next to yours holds a freighter that is practically falling apart. Standing next to it is a man with the grizzled look of an experienced captain.`
+			`	Given the sorry state of the ship he might be in need of assistance.`
+			choice
+				`	(Offer your help.)`
+				`	(Don't help him.)`
+					decline
+			`	The captain notices you approaching, and clearly guessing your intentions, speaks. "Oh don't worry, this ain't my ship."`
+			choice
+				`	"What are you doing with it then?"`
+					goto what
+				`	"Whose is it then?"`
+			`	"Well quite simply, it's nobody's."`
+			choice
+				`	"What are you doing with it then?"`
```

Comment:
I'd add another dialog option here for variety. It doesn't need to lead to another label, but just something for roleplay purposes.